src/modules/person/api/person.controller.ts

```diff
@@ -526,6 +528,21 @@ export class PersonController {
             );
         }
 
+        if (result.personalnummer) {
+            const userLocks: UserLock[] = await this.userLockRepository.findByPersonId(params.personId);
+            const koperslock: UserLock | undefined = userLocks.find(
+                (lock: UserLock) => lock.locked_occasion === PersonLockOccasion.KOPERS_GESPERRT,
+            );
+            if (koperslock) {
+                await this.keycloakUserService.updateKeycloakUserStatus(
+                    params.personId,
+                    result.keycloakUserId!,
+                    koperslock,
+                    false,
+                );
+            }
+        }
+
```

Comment:
Could we move this into the `PersonRepository.updatePersonMetadata`? If we update the kopers somewhere else the users would not be unlocked (I don't know why we would do that, but could happen in the future. Maybe bulk-operations or something like that)

Reply:
all done